js/main.js

```diff
@@ -0,0 +1,103 @@
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  homeBtn.classList.remove('active');
+}));
+
+// Speaker Data
+
+const speakers = {
+  speakerData: [
+    {
+      img: 'img/boopmeep.jpg',
+      name: 'BoopMeep',
+      work: 'Mage main pve player, content creator and streamer',
+      bio: 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Inventore, magni laudantium? Ipsam cum voluptas accusamus!',
+    },
+    {
+      img: 'img/Drjayfisto.jpg',
+      name: 'Dr-jayfisto',
+      work: 'Mage main pve player, content creator, theory crafter and streamer',
+      bio: 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Inventore, magni laudantium? Ipsam cum voluptas accusamus!',
+    },
+    {
+      img: 'img/preheat.jpg',
+      name: 'Preheat',
+      work: 'Mage main pve player, theory crafter, content writer at WoWHead, and streamer',
+      bio: 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Inventore, magni laudantium? Ipsam cum voluptas accusamus!',
+    },
+    {
+      img: 'img/mitchjones.jpg',
+      name: 'Mitchjones',
+      work: 'Mage main pvp player and streamer',
+      bio: 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Inventore, magni laudantium? Ipsam cum voluptas accusamus!',
+    },
+    {
+      img: 'img/Venruki.jpg',
+      name: 'Venruki',
+      work: 'Mage main pvp player, theroy crafter, co-host at AWC tournament and streamer',
+      bio: 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Inventore, magni laudantium? Ipsam cum voluptas accusamus!',
+    },
+    {
+      img: 'img/xaryu.jpg',
+      name: 'Xaryu',
+      work: 'Fire mage main pvp player, played WOW since 2004, content creator, AWC tournament champion, and streamer',
+      bio: 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Inventore, magni laudantium? Ipsam cum voluptas accusamus!',
+    },
+  ],
+};
+
+// Speaker Elements & Styles
+
+let speakerDiv = '';
+
+speakers.speakerData.forEach((speaker) => {
+  speakerDiv += `<div class="card">
+  <div class="speakerImg"><img width="150" src=${speaker.img} alt=${speaker.name}></div>
+  <div class="aboutSpeaker">
+      <h4 class="name">${speaker.name}</h4>
+      <h6 class="work"><i>${speaker.work}</i></h6>
+      <div class="underline"></div>
+      <p class="aboutText">${speaker.bio}</p>
+  </div>
+</div>`;
+});
+
+document.querySelector('#speakerCards').innerHTML = speakerDiv;
+
+const more = document.querySelector('.more');
+const close = document.querySelector('.close');
+const spClass3 = document.querySelector('.card:nth-child(3)');
+const spClass4 = document.querySelector('.card:nth-child(4)');
+const spClass5 = document.querySelector('.card:nth-child(5)');
+const spClass6 = document.querySelector('.card:nth-child(6)');
+
+more.addEventListener('click', () => {
+  spClass3.style.display = 'flex';
+  spClass4.style.display = 'flex';
+  spClass5.style.display = 'flex';
+  spClass6.style.display = 'flex';
```

Comment:
I really loved the "Show more" functionality! 🚀 Kudos for this one!

If you're curious, I have a challenge for you: If you see, you're using fixed numbers (3-6) what if you want to add more speakers in the future? you would have to not only change your JavaScript object with the information, but you would have to update this code too.

What if you create a function that gets all the DOM speaker elements and toggles this display/hide functionality without having to rely on fixed numbers? It would be great if you try to achieve it once you get more into JavaScript in the next module! 🤓 🛠️

Reply:
Thank you for all of your efforts to review these details 🤩.